Sources/SwiftFrameCore/Extensions/NSError+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+extension NSError {
+
+    convenience init(code: Int = 1, description: String) {
+        self.init(domain: "com.kayak.SwiftFrame", code: code, userInfo: [NSLocalizedDescriptionKey: description])
+    }
+
+}
+
+public func ky_executeSafely<T>(_ work: () throws -> T) -> T {
```

Comment:
Nit: Naming is a bit odd here IMHO. If it catches the error, it will kill the process which seems as "safe" as doing `try!` / not catching the exception. I like the `...OrExit` naming that you used in other methods. Maybe this could be used here as well.